### PR TITLE
Forces hardware rendering for all connecting clients.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
:cl:
experiment: The game will now force hardware rendering on for all clients.
/:cl:

### CODE REVIEW

**Comment by SgtxRootbeer** [comment]
No, one time, I had an issue with the latest gfx drivers not liking /tg/ code. My only solution was to disable hardware rendering temporarily.

**Comment by optimumtact** [comment]
if you disable hardware rendering the game will literally fail to render Rootbeer

**Comment by MrStonedOne** [comment]
and it can be abused to disable some parts of darkness

**Review by Cyberboss — APPROVED** [review]

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -195,11 +195,11 @@ var/next_external_rsc = 0
 	sethotkeys(1)						//set hoykeys from preferences (from_pref = 1)
 
 	. = ..()	//calls mob.Login()
-
+
```
Comment (JJRcop) [inline code/modules/client/client_procs.dm]: yuck

Comment (JJRcop) [inline code/modules/client/client_procs.dm]: Pointless whitespace

**Review by JJRcop — CHANGES_REQUESTED** [review]

File: code/modules/client/client_procs.dm (hunk 1 above)
Comment (MrStonedOne) [inline code/modules/client/client_procs.dm]: no where near as pointless as the commit to remove it would be.

Comment (optimumtact) [inline code/modules/client/client_procs.dm]: bitch you added it in the first place

Comment (MrStonedOne) [inline code/modules/client/client_procs.dm]: Ya, i meant to copy paste the line to here, but when i tabbed back i put it elsewhere.

It happens,

**Comment by optimumtact** [comment]
maybe a test merge?

**Pull request merged**